src/main/java/io/stargate/sgv2/jsonapi/service/operation/tables/TableRowProjection.java

```diff
@@ -73,7 +73,14 @@ public DocumentSource documentSource(Row row) {
             "Column '%s' has unsupported type '%s'", columnName, column.getType().toString());
       }
       try {
-        result.put(columnName, codec.toJSON(objectMapper, row.getObject(i)));
+        final Object columnValue = row.getObject(i);
+        // We have a choice here: convert into JSON null (explicit) or drop (save space)?
```

Comment:
Added a comment since we have discussed possibility of suppressing nulls to save space: whether it'd be done here or as post-processing is an open question; at first we just want to avoid NPE.